frontend/src/containers/ConfigStep/DateRangePicker/style.tsx

```diff
@@ -1,23 +1,62 @@
+import { BasicButton } from '@src/components/Common/Buttons';
 import { DatePicker } from '@mui/x-date-pickers';
 import { styled } from '@mui/material/styles';
+import { Button } from '@mui/material';
 import { theme } from '@src/theme';
+
+export const DateRangePickerGroupContainer = styled('div')({
+  border: `${theme.main.cardBorder}`,
+  borderRadius: '0.25rem',
+  padding: '1rem 1.5rem 1.5rem 1.5rem',
+});
+
+export const TitleContainer = styled('div')({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+});
+
+export const StyledFeaturedRangePickerContainer = styled('div')({
+  '&:first-of-type>.range-picker-row': {
+    marginTop: '1rem',
+  },
+});
+
 export const StyledDateRangePickerContainer = styled('div')({
   display: 'flex',
   width: '100%',
   justifyContent: 'space-between',
   gap: '1.5rem',
-  marginTop: '1rem',
   [theme.breakpoints.down('sm')]: {
     flexDirection: 'column',
   },
+  position: 'relative',
+  paddingBottom: '2.5rem',
 });
 
 export const StyledDateRangePicker = styled(DatePicker)({
   width: '50%',
   button: {
     margin: '0.75rem',
   },
+  marginTop: '-0.4rem',
   [theme.breakpoints.down('sm')]: {
     width: '100%',
   },
 });
+
+export const AddButton = styled(Button)({
```

Comment:
Maybe duplicated component with the `add pipeline` of the pipeline in metrics page

Reply:
Yeah, quite similar but not 100% the same, structure and margin vary.